src/main/java/htsjdk/samtools/cram/ref/EnaRefService.java

```diff
@@ -16,6 +16,8 @@
     private static final int HTTP_NOT_FOUND = 404;
     private static final int HTTP_INTERNAL_SEVER_PROBLEM = 500;
     private static final int HTTP_CONNECTION_TIMEOUT = 522;
+    private static final int HTTP_MOVED_PERMANENTLY = 301;
```

Comment:
It seems that ```HttpURLConnection``` defined most if not all this constants since you are changing this bit perhaps you could go ahead and use those instead of defining our own.